### PR TITLE
upgrade to node 16

### PR DESCRIPTION
**Pull request by stevesims**
bump time!

### CODE REVIEW

File: packages/govuk-react/package.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
   },
   "peerDependencies": {
     "react": ">=16.2.0",
-    "styled-components": "^4.1.3"
+    "styled-components": ">=5.1"
```
Comment (penx) [inline packages/govuk-react/package.json:66]: huh how did I miss that

**Review by penx — DISMISSED** [review]

File: packages/govuk-react/package.json (hunk 1 above)
Comment (stevesims) [reply packages/govuk-react/package.json:66]: dunno - I spotted it whilst one of my many attempted builds was happening :grin:

File: components/header/src/index.js [hunk 2]
```diff
@@ -73,4 +73,5 @@ Header.propTypes = {
 export default deprecate(Header, 'please use the Heading component instead');
 
 export { Header as DocumentedHeader };
+// eslint-disable-next-line import/no-cycle
 export { H1, H2, H3, H4, H5, H6 } from './presets';
```
Comment (stevesims) [inline components/header/src/index.js:77]: the point of this linting warning is that `presets` is importing this file, thus this is a circular dependency

ideally we'd work around this by having the `Header` component in a separate `atom` file, and import that into the `index` file, allowing the `presets` file to import from the `atom` file, thus ensuring there is no circularity

unfortunately this doesn't work with our documentation generation system.  attempting to do that results in empty documentation

another possibility would be to define H1-H6 inside this file, however that too is incompatible with our documentation generation system, as it results in multiple components being defined inside this file

File: components/error-summary/src/__snapshots__/test.js.snap [hunk 3]
```diff
@@ -93,6 +93,7 @@ exports[`error summary matches the ErrorSummary snapshot: error summary 1`] = `
                 escapeHtml={false}
                 linkRenderer={[Function]}
                 mb={3}
+                parserOptions={Object {}}
```
Comment (penx) [inline components/error-summary/src/__snapshots__/test.js.snap:96]: odd, any idea what this is?

File: components/file-upload/src/index.js [hunk 4]
```diff
@@ -93,10 +93,13 @@ FileUpload.propTypes = {
     active: PropTypes.bool,
     dirty: PropTypes.bool,
     dirtySinceLastSubmit: PropTypes.bool,
+    // eslint-disable-next-line react/forbid-prop-types
```
Comment (penx) [inline components/file-upload/src/index.js:96]: these should be PropTypes.node I think - please could you open an issue to review these?

File: components/header/src/index.js (hunk 2 above)
Comment (penx) [reply components/header/src/index.js:77]: please can you open an issue to review this in 0.9?

File: components/tabs/src/fixtures.js [hunk 5]
```diff
@@ -226,16 +238,16 @@ const arrProposedBabel = [
...
-
-  static propTypes = sharedPropTypes;
-
   state = { tabIndex: this.props.defaultIndex };
 
   setTabIndex = setTabIndex;
 
   handleClick = handleClick;
 
+  static propTypes = sharedPropTypes;
```
Comment (penx) [inline components/tabs/src/fixtures.js:247]: why move these?

**Comment by penx** [comment]
approved other than comments

File: components/error-summary/src/__snapshots__/test.js.snap (hunk 3 above)
Comment (stevesims) [reply components/error-summary/src/__snapshots__/test.js.snap:96]: nope - snapshots be crazy bro!

(it's probably the result of a minor bump in *one* of the modules updated in this PR)

File: components/tabs/src/fixtures.js (hunk 5 above)
Comment (stevesims) [reply components/tabs/src/fixtures.js:247]: linting was saying that `propTypes` had to go after `handleClick`, and the `defaultProps` after `propTypes`.

we seem to have ended up with slightly stricter linting rules with the updated yarn.lock file

File: components/file-upload/src/index.js (hunk 4 above)
Comment (stevesims) [reply components/file-upload/src/index.js:96]: added #794

File: components/error-summary/src/__snapshots__/test.js.snap (hunk 3 above)
Comment (penx) [reply components/error-summary/src/__snapshots__/test.js.snap:96]: maybe react markdown got an upgrade and this is a default prop

**Review by penx — APPROVED** [review]

**Pull request merged**